vizro-core/docs/pages/development/contributing.md

```diff
@@ -47,13 +47,19 @@ If no example is specified (`hatch run example`) then the [default example](http
 
 ## Testing
 
-Tests are handled using the [`pytest`](https://docs.pytest.org/) framework, and test environments are managed by Hatch. To run all tests, run
+Tests are handled using the [`pytest`](https://docs.pytest.org/) and [`jest`](https://jestjs.io/) frameworks, and test environments are managed by Hatch. To run all tests, run
 
 ```console
 hatch run test
 ```
 
-To run only unit tests, run `hatch run test-unit` and for integration tests only, run `hatch run test-integration`. Note that the latter executes all examples in separate testing sessions due to [this](https://github.com/mckinsey/vizro/issues/10) issue by providing the `-k` tag per example.
+To run only unit tests, run `hatch run test-unit`, and for integration tests only run `hatch run test-integration`.
+
+To run only jest unit tests for javascript functions, run `hatch run test-js`.
+Note that Node.js is required to run tests written in jest framework. If you don't have Node.js installed, guidelines on how to install Node.js will appear when you run the command: `hatch run test-js`.
+Otherwise, if Node.js is installed, then the same command (`hatch run test-js`) runs jest unit tests.
```

Comment:
```suggestion
To run only jest unit tests for javascript functions, run `hatch run test-js`.
Note that Node.js is required to run tests written in the jest framework. If you don't have `Node.js` installed, guidelines on how to install Node.js will appear when you run the command: `hatch run test-js`.
Otherwise, if `Node.js` is installed, then the same command (`hatch run test-js`) runs jest unit tests.
```